src/models.rs

```diff
@@ -609,6 +609,7 @@ pub struct Database {
     pub disk_size_in_gib: u32,
     pub database_instance_type: String,
     pub database_disk_type: String,
+    pub database_disk_encryption: bool,
```

Comment:
Why prefix the field with database_ ? We are already in the database object

Reply:
To have consistency with others. But I agree we can remove it as it's already in databse